src/deferred.js

```diff
@@ -31,7 +31,7 @@ function adoptValue( value, resolve, reject ) {
 
 			// Support: Android 4.0 only
 			// Strict mode functions invoked without .call/.apply get global-object context
-			resolve.call( undefined, value );
+			resolve.apply( undefined, [ value ].slice( noValue ) );
```

Comment:
This looks like "clever code", this passing booleans to slice. I guess that's done to reduce size (a regular if-else would be larger) but at least an explanatory comment would be useful.